services/websocket/issue_comment_notifier.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package websocket
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	issues_model "code.gitea.io/gitea/models/issues"
+	user_model "code.gitea.io/gitea/models/user"
+	"code.gitea.io/gitea/services/pubsub"
+)
+
+func (n *websocketNotifier) DeleteComment(ctx context.Context, doer *user_model.User, c *issues_model.Comment) {
+	d, err := json.Marshal(c)
+	if err != nil {
+		return
+	}
+
+	n.pubsub.Publish(ctx, pubsub.Message{
```

Comment:
Maybe the topic should still contain the issue and index. And it also includes the actions like delete/create/update. Because I think different pages of the same clients should not share the same web socket client.